docs/PyOp.md

```diff
@@ -0,0 +1,71 @@
+# Python Operator 
+To facilitate Python coders on model developing, onnxruntime provides a way to invoke operators implemented in Python.
+
+## Usage
+Step 1, build onnxruntime with“--config Release --enable_language_interop_ops --build_shared_lib”and override existing onnxruntime binary with the latest, then copy onnxruntime_pywrapper.dll or libonnxruntime_pywrapper.so or libonnxruntime_pywrapper.dylib to the path where onnxruntime binary is placed.
```

Comment:
There is also a flag to avoid the build to install a different version of numpy and build with the current one. Maybe that's Worth mentioning in this case.

Reply:
OK, but which flag you mean exactly?

Reply:
--numpy_version= with no value

Reply:
Added a note for that. Let's if it is good enough.